src/index.html

```diff
@@ -1,13 +1,47 @@
-<!doctype html>
+
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+        <div class="card__image">
+          <img src="./images/imac.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        </div>
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <h2 class="product-code">Product code: 195434</h2>
```

Comment:
```suggestion
        <p class="product-code">Product code: 195434</p>
```
There is a regular text, so there is no need to use the tag for titles